src/sourceMaps/sourceMapUtils.ts

```diff
@@ -64,11 +64,16 @@ export function getComputedSourceRoot(sourceRoot: string, generatedPath: string,
     return absSourceRoot;
 }
 
+let aspNetFallbackCount = 0;
```

Comment:
What is the significance of this variable?

Reply:
We use it to count how many times the "fallback" bevahior is happening. This will also tell us how many users are using this fix. We send it as telemetry.

Reply:
Wouldn't this count just reflect on the number of files in the user's project? What I implied was if storing just yes/no if this feature was needed be enough for telemetry purposes.

Reply:
It's the same effort to send yes vs. no than a number.
It could be useful to know this number a certain point (maybe we see that the debugger is very slow when this number gets bigger, so there might be a performance issue in this fix)